src/metabase/events/recent_views.clj

```diff
@@ -41,6 +42,7 @@
                    ;; we don't want to count pinned card views
                    ((complement #{:collection :dashboard}) context))
           (recent-views/update-users-recent-views! user-id model model-id))
-        (record-view! model model-id user-id metadata)))
+        (let [details (audit-log/model-details (merge object metadata) (keyword (name topic)))]
```

Comment:
To match the audit-log :details and the view-log :metadata, I've modified recent-views here.

@noah, I don't need a full review on this yet but I want to get a gut check from you if this is an ok way to achieve that.

Reply:
looks fine to me